Automattic/jetpack
# Publicize: Sync changes made to wpcom's admin-plugins/publicize.php file.

To sync the changes in D82384-code
#### Changes proposed in this Pull Request:

* Sync the changed made in the wpcom patch. 

#### Jetpack product discussion


p1656518552143489/1656511761.126559-slack-C02JJ910CNL

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Disable the publicize module and make sure the publicize related elements don't show up on the block editor. 
* Enable the publicize module and make sure the publicize elements show up on the block editor and you're able to publicize as usual.
